src/js/classes/Table.js

```diff
@@ -245,6 +269,32 @@ export class RankedTable {
   }
 
   render() {
+    // set up search bar
+    const searchMenu = this.container.getElementsByClassName("menu")[0];
+    const searchOptions = this.data.flatMap((row) =>
+      row.flatMap((value, i) =>
+        this.searchCols[i] ? [value.toLocaleString()] : []
+      )
+    );
+    // Current behavior is to alphabetically sort all options,
+    // potentially mixing values from different columns
+    // TODO: Consider dividing values by column
+    searchOptions.sort();
+    searchMenu.textContent = "";
+    searchOptions.forEach(searchOption => {
+      const element = document.createElement("div");
+      element.className = "item";
+      element.innerText = searchOption;
+      searchMenu.appendChild(element);
+    });
+    const searchInput = this.element.parentNode.getElementsByTagName("input")[0];
```

Comment:
You already have this saved as `this.container`, right? So you shouldn't have to do `this.element.parentNode`.

Reply:
Done